app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
The swedish was there (hardcoded in the views/messages), its the english that is added